crates/anvil-polkadot/src/api_server/server.rs

```diff
@@ -945,6 +1014,77 @@ impl ApiServer {
         Ok(())
     }
 
+    // ----- Wallet RPCs
+    async fn sign(&self, address: Address, content: impl AsRef<[u8]>) -> Result<String> {
+        node_info!("eth_sign");
+        let signature = alloy_primitives::hex::encode(
+            self.wallet.sign(ReviveAddress::from(address).inner(), content.as_ref())?,
+        );
+        Ok(format!("0x{signature}"))
```

Comment:
nit: use alloy_primitives::hex::encode_prefixed (applies to sign_typed_data_v4) as well